### ISSUE TITLE
bind-fields on changed elements

### ISSUE THREAD

**Issue by deg** [reagent-project/reagent-forms#30]
TL;DR I'm trying to use bind-fields for input validation on elements that are not always on the page. I can't get it to work.

Details: My page is a single-page app with a menu bar in the header that controls what is displayed on the rest of the page. One of the layouts is mostly copied from forms-example here. When I include it as a one of the page bottoms, the field validations no longer work.

I'm not sure what is supposed to work at this stage, so I'm wondering if:
- (a) this is supposed to work, but I've done something stupid;
- (b) this is not supposed to work yet;
- (c) this doesn't really work yet but there is a known workaround; or
- (d) this has been made to work in  a very recent commit to reagent-forms; later than what is on clojars. (I've not looked closely enough yet, but some of the commit comments form the last day seem to suggest this)

My current code is way too dirty to ask anyone to read it but, fwiw, is at https://github.com/deg/chestnut1. (Pay little attention to the name. It began as an attempt to move Chestnut from Om to Reagent).

Thanks, and apologies that this is closer to a cry-for-help than a real issue report.
